### ISSUE TITLE
skip duplicates with eth-brownie and pytest-dependency

### ISSUE THREAD

**Issue by Senopiece** [renanivo/pytest-testdox#67]
When using both eth-brownie and pytest-testdox in the same project it occurs that if a test is skipped, the message will be displayed twice.

> stdout example:
```
test_US_001.py                                                                                 

 ✗ AC-001-01
                                                                         [  5%]
test_US_002.py                                                                                 

 » AC-002-01 <- as AC-002-01 depends on passing AC-001-01
 » AC-002-01 <- unnecessary duplicate
```

**Comment by renanivo** [comment]
Hi, @Senopiece. Thanks for submitting this issue. 

I have a question about the scope of it:  How the `AC-002-01` dependency of `AC-002-01` is declared?

**Comment by Senopiece** [comment]
Here you can see the simplest showcase: https://github.com/Senopiece/test

**Comment by renanivo** [comment]
Hi @Senopiece. I tried to reproduce the issue, but I got the error:
```
INTERNALERROR> FileNotFoundError: [Errno 2] No such file or directory: 'ganache-cli'
```

Looks like this `ganache-cli` is required to run the Eth-Brownie. I was looking at the pytest hooks registered in Eth Brownie and I couldn't find where is the problem. As it is the only plugin conflicting with pytest-testdox (so far) and I couldn't find where is the issue, I will not accept the Pull Request -- Because I would not be able to know if a new feature brings the issue back.

I'm closing this issue and PR for now, but if you have any new finding, feel free to comment.
